hibernate-core/src/main/java/org/hibernate/proxy/AbstractLazyInitializer.java

```diff
@@ -115,7 +115,7 @@ else if ( isConnectedToSession() ) {
 					entityName,
 					id
 				);
-				throw new HibernateException( "illegally attempted to associate a proxy with two open Sessions" );
+				throw new HibernateException( "illegally attempted to associate proxy (" + entityName + "/" + id + ") with two open Sessions" );
```

Comment:
The standard way we refer to entity "instances" in log messages is `<entity-name>#<id-value>`.  We should be consistent

Reply:
Changed output: https://github.com/hibernate/hibernate-orm/pull/2248